src/main/java/frc/robot/climb/ClimbHw.java

```diff
@@ -15,7 +15,7 @@ public class ClimbHw extends Climb {
   private TalonFXMotor climbMotor;
   TalonFX rawMotor;
   PositionVoltage positionSetter;
-  static final double CLIMB_END_MEASURE = 0.31;
+  static final double CLIMB_END_MEASURE = 0.324;

```

Comment:
This is a number you have to get from the cancoder on the climb arm.  It is the MAX the arm allows.  To recalibrate it, let the string all the way out, that is zero.  Reset the zero in Phoenix Tuner.  Then push the climber all the way in.  Do a status update (I forget the name) that gets the current value of the sensor, and put it here.  Maybe take a tiny bit off, but each 0.01 makes a BIG difference (it is in rotations, so each 0.01 is 3.6*)